Integrations/integration-Panorama.yml

```diff
@@ -3492,7 +4078,42 @@ script:
     - name: password
       description: Password for Panorama, needed for dlp-pcap only.
     description: Returns a list of pcap names
+  - name: panorama-register-ip-tag
+    arguments:
+    - name: tag
+      required: true
+      description: Tag to register IPs to
+    - name: IPs
+      required: true
+      description: IPs to register
+      isArray: true
+    - name: persistent
```

Comment:
why 0,1 and not true false? is that something from the API

Reply:
yes. i can twik the args and context values to bollean if that's more readable